test/modellib/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+
+add_onnx_mlir_library(ModelLib
+  ConvModel.cpp
+  GemmModel.cpp
+  GRUModel.cpp
+  MatMulModel.cpp
+  ModelLib.hpp
+  ModelHelper.cpp
+  ModelHelper.hpp
```

Comment:
Don't include header files in the list of sources - they should be included in the source files instead.

Reply:
I assume from your comment that there should just be removed from the list.